src/dev/riscv/LupV.py

```diff
@@ -94,35 +94,35 @@ class LupV(Platform):
     clint = Param.Clint(Clint(pio_addr=0x2000000), "CLINT")
 
     # PLIC
-    plic = Param.Plic(Plic(pio_addr=0xc000000), "PLIC")
+    pic = Param.Plic(Plic(pio_addr=0xc000000), "PLIC")
 
     # LUPIO RTC
     lupio_rtc = Param.LupioRTC(LupioRTC(pio_addr=0x20004000), "LupioRTC")
 
-    # Uart
-    uart = Uart8250(pio_addr=0x10000000)
+    # LUPIO TTY
+    lupio_tty = LupioTTY(pio_addr=0x20007000)
+
     # Int source ID to redirect console interrupts to
-    # Set to 0 if using a pci interrupt for Uart instead
-    uart_int_id = Param.Int(0xa, "PLIC Uart interrupt ID")
+    lupio_tty_int_id = Param.Int(0xb, "PLIC LupioTTY interrupt ID")
```

Comment:
Why not keep the same interrupt ID as the uart? (it's a question)

Reply:
No good reason, except when I was examining the device tree in QEMU I think the interrupt id for lupio-tty was 0xb but I just changed it back to 0xa